sdks/python/apache_beam/io/parquetio_test.py

```diff
@@ -296,8 +296,10 @@ def test_sink_transform_int96(self):
               path, self.SCHEMA96, num_shards=1, shard_name_template='')
 
   def test_sink_transform(self):
-    with tempfile.NamedTemporaryFile() as dst:
-      path = dst.name
+    fd, path = tempfile.mkstemp()
```

Comment:
There was problem on windows since `with tempfile.NamedTemporaryFile() as dst` provide `tempfile` which is already opened. On windows any opened file seems to be read only which coused PermissionError. I know this workaround is ugly but it the best way I found to do it.
Is separate commit with JIRA will not be lost after merging?

Reply:
You can use  `tempfile.TemporaryDirectory()` to generate a fresh temp directory and use any name as a file name within this directory with the test.

Reply:
I updated it with `tempfile.TemporaryDirectory()`, looks much more better and intuitive now. Thanks a lot!